### PR TITLE
updated mocha commit

### LINKED ISSUE #181: Update Mocha to the latest version
As of this writing, [Mocha](https://mochajs.org/), our test runner, is a major version behind. We should update it with `npm install mocha@latest --save-dev` and verify that everything works.

### PR DESCRIPTION
**Pull request by BeardyGod**
Fix #181 
As asked, I hae updated mocha to latest version (8.0.1).
I ran npm test and passed all the test.

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.24.0 / 2020-06-17
```
Comment (EvanHahn) [inline HISTORY.md:1]: This is an internal-only change so we don't need to update the public changelog. `git checkout master HISTORY.md` should revert this change.

File: README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ npm install humanize-duration
 With `require` (like in Node or with common build systems):
 
 ```js
-const humanizeDuration = require("humanize-duration");
+const humanizeDuration = require('humanize-duration');
```
Comment (EvanHahn) [inline README.md:22]: I don't think the readme should've changed as part of this pull request, except for adding your name to the credits. Could you revert?

Comment (BeardyGod) [reply README.md:22]: ok. I'm on it
